src/main/java/Task.java

```diff
@@ -0,0 +1,36 @@
+import java.util.Arrays;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    private static int totalTask = 0;
+    private static Task[] taskList = new Task[100];
```

Comment:
So every task has a reference to the taskList? Then it is irrelevant to store a Task[] in Duke. Instead, static setters and getters could be used.

Reply:
Thanks for the great idea. Will implement it.